nncf/common/graph/graph.py

```diff
@@ -607,12 +607,12 @@ def get_graph_for_structure_analysis(self, extended: bool = False) -> nx.DiGraph
                 else:
                     attrs_edge["style"] = "solid"
                 label["shape"] = edge[NNCFGraph.ACTIVATION_SHAPE_EDGE_ATTR]
+                label[
+                    "ports"
+                ] = f"{edge[NNCFGraph.OUTPUT_PORT_ID_EDGE_ATTR]}\u2192{edge[NNCFGraph.INPUT_PORT_ID_EDGE_ATTR]}"
```

Comment:
I understand that you tried to make the representation prettier, but even in 2023 there's no guarantee that every tool can handle Unicode well enough. Use `->` to represent an arrow, that would be safer.